tests/integration_tests/test_kernel_commandline_match.py

```diff
@@ -11,7 +11,24 @@
 log = logging.getLogger("integration_testing")
 
 
-def override_kernel_cmdline(ds_str: str, c: IntegrationInstance) -> str:
+def restart_cloud_init(c):
+    client = c
```

Comment:
nit:non-blocking why not just pass in a variable named client and use that.

Reply:
since `client` is a fixture, I didn't want to use that name because 1) I don't know if a fixture would be invoked (non-test functions can get fixtures invoked, for example a fixture that invokes other fixtures) and 2) even if 1 doesn't occur, it might confuse a reader who might assume that it does. This is more clear in my mind, though I guess I could have just picked a different variable name.